imkg/.htaccess

```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://memes4.science [R=302,L]
```

Comment:
Will this resolve soon?  We prefer if redirects work from the start.  I do look forward to seeing what this project is. :-)